source/particle/world.cc

```diff
@@ -1276,6 +1288,7 @@ namespace aspect
       generator.reset(Generator::create_particle_generator<dim> (prm));
       if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(generator.get()))
         sim->initialize_simulator (this->get_simulator());
+      generator->parse_global_parameters(prm);
```

Comment:
Same here, the call to `Interface::parse_parameters` should likely come from the derived classes via the call to `generator->parse_parameters(prm);` below